docker-compose.yml

```diff
@@ -40,8 +40,11 @@ services:
       ENTITY_ID: "${ENTITY_ID}"
       SP_PUBLIC_CERT: "${SP_PUBLIC_CERT}"
       SP_PRIVATE_KEY: "${SP_PRIVATE_KEY}"
+      BAD_HOST: "${BAD_HOST}"
+      OVERIDE_HOST: "${OVERIDE_HOST}"
     volumes:
       - ./app:/opt/app/openlxp-xds
+      - ${CERT_VOLUME}:/etc/ssl/certs/ca-certificates.pem
```

Comment:
Just curious on what the secrets value will be for this, if you do have a solution we can go ahead and add this in all components.